libs/src/sdk/api/generator/templates/typescript-fetch/models.mustache

```diff
@@ -1,6 +1,6 @@
-// @ts-nocheck
 /* tslint:disable */
 /* eslint-disable */
+// @ts-nocheck
 {{>licenseInfo}}
```

Comment:
unfortunately still need the `// @ts-nocheck` even on the new version 😢 . It's the only edit I made to the default template (here and in the apis one as well).